diff --git a/arch/arm64/boot/dts/freescale/Makefile b/arch/arm64/boot/dts/freescale/Makefile
index cc32798..9482065 100644
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -132,7 +132,8 @@ dtb-$(CONFIG_ARCH_FSL_IMX8MM) += fsl-imx8mm-evk.dtb \
                                 fsl-imx8mm-evk-revb.dtb \
                                 fsl-imx8mm-evk-revb-rm67191.dtb \
                                 fsl-imx8mm-ddr4-evk-rm67191.dtb \
-                                fsl-imx8mm-ddr4-qca9377-evk.dtb
+                                fsl-imx8mm-ddr4-qca9377-evk.dtb \
+                                imx8mmevk.dtb
 dtb-$(CONFIG_ARCH_FSL_IMX8MN) += fsl-imx8mn-ddr4-evk.dtb \
                                 fsl-imx8mn-ddr4-evk-rpmsg.dtb \
                                 fsl-imx8mn-ddr4-evk-rm67191.dtb \
